app/crud.py

```diff
@@ -46,7 +46,8 @@ def get_progress(db: Session, election_ref: str, token: str) -> schemas.Progress
         raise errors.UnauthorizedError("Wrong election ref")
     
     # Check we can update the election
-    if not payload["admin"]:
+    # Use .get() for a safe check. If "admin" key is missing, it returns None (which is falsy).
```

Comment:
C'est la même chose que `not playload["admin"]` non? 👀   
Si ça perturbe et que tu préfères, il y a également la synthaxe: `"admin" not in payload`

Reply:
ouki, ça marche 👍 

Merci pour les explications ;)